longtask-timing/longtask-attributes.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE HTML>
+<meta charset=utf-8>
+<title>LongTask Timing: validate long task attributes</title>
+<body>
+
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<h1>Long Task Attributes</h1>
+<div id="log"></div>
+<script>
+  async_test(function (t) {
+    var observer = new PerformanceObserver(
+        t.step_func(function (entryList) {
+            var entries = entryList.getEntries();
+            assert_equals(entries.length, 1,
+                "Exactly one entry is expected.");
+            var longtask = entries[0];
+            assert_equals(longtask.entryType, "longtask");
+            assert_equals(longtask.name, "same-origin-self");
+            assert_greater_than(longtask.duration, 50);
+            assert_equals(longtask.startTime, Math.floor(longtask.startTime),
+                "startTime expected to have 1 miillisecond granularity");
```

Comment:
Rounding sounds better, but looking at existing code for "clamping resolution" I'm finding floor.
Filed crbug.com/681718 to track this.